src/wiki/development/build-instructions.md

```diff
@@ -85,7 +85,7 @@ makedeb -s
 
 The .deb will be located in the directory the repo was cloned in.
 
-### Building an .rpm
+### Building an .rpm for Fedora
 
 Build dependencies are automatically installed using `DNF`, however, you will also need the `rpmdevtools` package (on Fedora),
```

Comment:
```suggestion
Build dependencies are automatically installed using `DNF`, however, you will also need the `rpmdevtools` package,
```
Since this section is fedora only now

Reply:
ah thanks!